paddle/infrt/dialect/tensorrt/trt_exec.cc

```diff
@@ -32,17 +32,34 @@ int main(int argc, char** argv) {
   mlir::MLIRContext* context = infrt::Global::getMLIRContext();
   auto module = infrt::dialect::LoadMlirFile(input_file.c_str(), context);
 
+  std::cout << "Input IR dump:\n";
```

Comment:
Remove this, no need to display the input IR.

The llvm lit will exemine the stdout, so avoid print unnecessary infomation to the stdout.